src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1140,13 +1699,13 @@ mod tests {
         should_deploy: bool,
     }
     impl TestRamAirTurbineController {
-        fn new() -> Self {
+        fn _new() -> Self {
```

Comment:
Why do you want an underscore here?

Reply:
Forced to add _ when unused, but means I could delete all of this so I just cleaned everything up now thanks :)